exdb/models.py

```diff
@@ -1,3 +1,93 @@
 from django.db import models
+from django.utils.timezone import now
 
-# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class SubType(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Type(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Audience(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Organization(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Keyword(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Experience(models.Model):
+    STATUS_TYPES = (
+        ('dr', 'Draft'),
+        ('pe', 'Pending'),
+        ('ap', 'Approval'),
+        ('de', 'Denied'),
+        ('ad', 'Approved'),
+        ('co', 'Completed'),
+        ('ca', 'Cancelled')
+    )
+
+    author = models.ForeignKey(User)
+    planners = models.ManyToManyField(User, related_name='planner_set')
+    title = models.CharField(max_length=300)
+    description = models.TextField()
+    start_datetime = models.DateTimeField()
+    end_datetime = models.DateTimeField()
+    type = models.ForeignKey(Type)
+    sub_type = models.ForeignKey(SubType)
+    goal = models.TextField(null=True, blank=True)
+    keywords = models.ManyToManyField(Keyword)
+    audience = models.ForeignKey(Audience)
+    guest = models.CharField(max_length=300, null=True, blank=True)
+    guest_office = models.CharField(max_length=300, null=True, blank=True)
```

Comment:
`CharField`s don't really need to be nullable since the empty string is a valid input.